src/imageio/format/jpeg.c

```diff
@@ -460,6 +461,46 @@ size_t params_size(dt_imageio_module_format_t *self)
   return sizeof(dt_imageio_module_data_t) + sizeof(int);
 }
 
+void *legacy_params(dt_imageio_module_format_t *self, const void *const old_params,
+                    const size_t old_params_size, const int old_version, const int new_version,
+                    size_t *new_size)
+{
+  if(old_version == 1 && new_version == 2)
+  {
+    typedef struct dt_imageio_jpeg_t
```

Comment:
I have noticed this only now, this should be dt_imageio_jpeg_v1_t, not dt_imageio_jpeg_t.